studies/us_states_validation/cori_model.R

```diff
@@ -0,0 +1,98 @@
+# Set working directory to current file path
```

Comment:
is this file straight from the cobey lab repo?

Reply:
It is ~80% from the Cobey lab repo, with some of my adjustments to adapt it to our use case. I meant to add a comment with a link attributing it to them. Will include that in a future iteration.